### DISCUSSION TITLE
user keymap not respected

### DISCUSSION THREAD

**Discussion by rickywu** [ayamir/nvimdots#1042] [Q&A] [answered]
lua\user\keymap\core.lua

```
return {
	-- customize
    ["n|<leader>nt"] = require("keymap.bind").map_cr("NvimTreeToggle"):with_desc("ui: NT"),
}

```

lua\user\keymap\init.lua

```
return vim.tbl_extend(
	"force",
	require("user.keymap.core"),
	require("user.keymap.completion").plug_map,
	require("user.keymap.editor"),
	require("user.keymap.lang"),
	require("user.keymap.tool"),
	require("user.keymap.ui")
)
```
not work, anything missed?

**Comment by Jint-lzxy** [comment]
Can't seem to reproduce. Are there any more specific details u can provide or do I miss anything?

![info](https://github.com/ayamir/nvimdots/assets/50296129/0bf79060-3d07-4587-a32d-db735dbf01ff)

**Comment by rickywu** [comment]
![image](https://github.com/ayamir/nvimdots/assets/442398/b42a3918-d99a-4862-aa4e-2eb5e4c2979c)

I only have these two files under user/keymap

**Comment by Jint-lzxy** [comment, marked as answer]
> I only have these two files under user/keymap

You need to keep other files (e.g., `user.keymap.lang`) in this directory even if they do nothing b/c otherwise this `pcall` will fail silently:
https://github.com/ayamir/nvimdots/blob/36ae982eb70501b2adaff2c1020421be6d026ca2/lua/keymap/init.lua#L32
